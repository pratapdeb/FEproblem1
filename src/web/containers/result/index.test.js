import React from 'react';
import { Provider } from 'react-redux'
import Result from './index'
import configureStore  from 'redux-mock-store'
import renderer from 'react-test-renderer'

const mockStore = configureStore([])

describe('<DestinationCard/>', () => {

    let store 
    let component

    beforeEach(() =>  {
        store  = mockStore({})
        component = renderer.create(
            <Provider store={store}>
                <Result.WrappedComponent />
            </Provider>
        )
    })

    it('Matches the snapshot', () => {
        expect(component.toJSON()).toMatchSnapshot()
    })
})