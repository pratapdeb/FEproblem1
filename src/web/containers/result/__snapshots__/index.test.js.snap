// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DestinationCard/> Matches the snapshot 1`] = `
<div
  className="ant-result ant-result-info"
>
  <div
    className="ant-result-icon"
  >
    <span
      aria-label="exclamation-circle"
      className="anticon anticon-exclamation-circle"
      role="img"
    >
      <svg
        aria-hidden="true"
        data-icon="exclamation-circle"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm-32 232c0-4.4 3.6-8 8-8h48c4.4 0 8 3.6 8 8v272c0 4.4-3.6 8-8 8h-48c-4.4 0-8-3.6-8-8V296zm32 440a48.01 48.01 0 010-96 48.01 48.01 0 010 96z"
        />
      </svg>
    </span>
  </div>
  <div
    className="ant-result-title"
  >
    AI Falcon Found
  </div>
  <div
    className="ant-result-subtitle"
  >
    Plant: undefined
  </div>
  <div
    className="ant-result-extra"
  >
    <button
      className="ant-btn ant-btn-primary"
      onClick={[Function]}
      type="button"
    >
      <span>
        Go Home
      </span>
    </button>
  </div>
</div>
`;
